app/models/ingredient.rb

```diff
@@ -0,0 +1,6 @@
+class Ingredient < ApplicationRecord
+  validates :name, presence: true,
+    uniqueness: true,
+    format: { with: /\A[a-z][a-z\s]+[a-z]\z/,
+    message: "only allows underscore letters and whitespace" }
```

Comment:
presence is simpler and shorter than format, but validation depends on your assumption so this is your and your team choice.